server/api-manager.js

```diff
@@ -1,8 +1,8 @@
 /*eslint-disable comma-spacing */
 /*eslint-disable no-console*/
 
-const USERNAME = "";
-const PASSWORD = "";
+const USERNAME = "Siteowner";
```

Comment:
Everyone will just be using the defaults at this stage, so let's keep them here for convenience.

Reply:
Agreed